tests/crypto/test_keyring.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import msgpack
+from nkms.crypto.keyring import KeyRing
+
+
+class TestKeyRing(unittest.TestCase):
+    def setUp(self):
+        self.keyring = KeyRing()
+        self.msg = b'this is a test'
+
+    def test_signing(self):
+        signature = self.keyring.sign(self.msg)
+
+        try:
+            sig = msgpack.loads(signature)
+            self.assertTrue(1, len(sig[0]))     # Check v
```

Comment:
For my part, I'd be happy to have some more verbosity here.  What is this test *actually* doing?  As a reader, I don't have any reason to understand *why* 1, 32, and 32 are appropriate lengths for these three elements of `sig`.

Reply:
Btw that's a good point. Curves different from secp256k1 could have size different from 32 bytes

Reply:
Yep, I have this on my TODO for this code. I need to find a way to determine the size of the curve explicitly. I think I can do this with one of the components of the curve and some math.